pgx/src/datum/sql_entity_graph/mod.rs

```diff
@@ -64,6 +64,72 @@ pub trait ToSql {
     fn to_sql(&self, context: &PgxSql) -> eyre::Result<String>;
 }
 
+/// Represents a pointer to a function that can transform a SqlGraphEntity to a SQL string
```

Comment:
Nit: Not actually a pointer. It's just a type alias.

Reply:
I meant that the type represents a pointer to a function, but you're right, that's a semantic distinction that isn't really relevant to users of pgx. Any suggestions on how best to document its purpose? I suppose in practice this doesn't need to be publically documented per se, since users are not constructing a value of this type anywhere, it is just necessary for functions they intend to be targets of `#[to_sql(path::to::function)]` to have the same signature.

Reply:
How about like "The signature of a function that can transform ..."?